server/src/main/java/umm3601/Server.java

```diff
@@ -10,6 +10,12 @@
 public class Server {
     public static void main(String[] args) throws IOException {
 
+        // This users looks in the folder `public` for the static web artifacts,
+        // which includes all the HTML, CSS, and JS files generated by the Angular
+        // build. This `public` directory _must_ be somwhere in the classpath;
+        // a problem which is resolved in `server/build.gradle`.
+        staticFiles.location("/public");
```

Comment:
It shouldn't switch to port 9000 when running the production server because now the files are being hosted by Spark, i.e. port 4567, rather than being hosted by webpack-dev-server on port 9000. Dev mode should redirect us to port 9000 as that is where our files are hosted.